### PR TITLE
Remove dynamo db from test scripts

### PR DESCRIPTION
**Pull request by martyncolmer**
Removed dynamoDB and switched to testing with mongo db

To test
Check out and run 'yarn test'

### CODE REVIEW

File: eq-author-api/middleware/identification/index.test.js [hunk 1]
```diff
@@ -123,10 +123,10 @@ describe("auth middleware", () => {
         req.header.mockImplementation(() => `Bearer ${expected}`);
         await middleware(req, res, next);
         expect(req.user).toMatchObject({
-          id: expect.any(String),
+          //id: expect.any(String),
```
Comment (MattElcock) [inline eq-author-api/middleware/identification/index.test.js:126]:
```suggestion
```

File: eq-author-api/scripts/test.sh [hunk 2]
```diff
@@ -4,50 +4,41 @@ set -e
...
 REDIS_PORT=$(docker port $REDIS_CONTAINER_ID 6379 | cut -d ':' -f 2)
 
-echo "dynamo started at: $DYNAMO_HOST"
+#echo "dynamo started at: $DYNAMO_HOST"
 echo "redis start at: $REDIS_PORT"
 
 function finish {
   echo "killing docker..."
-  docker rm -vf $DYNAMO_CONTAINER_ID
+#  docker rm -vf $DYNAMO_CONTAINER_ID
```
Comment (MattElcock) [inline eq-author-api/scripts/test.sh:22]:
```suggestion
```

File: eq-author-api/scripts/test.sh [hunk 3]
```diff
@@ -4,50 +4,41 @@ set -e
...
-DYNAMO_USER_TABLE_NAME=test-author-users
 FIREBASE_PROJECT_ID=test-firebase-id
 
-echo "starting Dynamo docker..."
+echo "starting Redis docker..."
 
-DYNAMO_CONTAINER_ID=$(docker run -tid -P -e AWS_REGION=$AWS_REGION -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY amazon/dynamodb-local)
-DYNAMO_HOST=$(docker port $DYNAMO_CONTAINER_ID 8000)
+# DYNAMO_CONTAINER_ID=$(docker run -tid -P -e AWS_REGION=$AWS_REGION -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY amazon/dynamodb-local)
+# DYNAMO_HOST=$(docker port $DYNAMO_CONTAINER_ID 8000)
```
Comment (MattElcock) [inline eq-author-api/scripts/test.sh:12]:
```suggestion
```

File: eq-author-api/scripts/test.sh [hunk 4]
```diff
@@ -4,50 +4,41 @@ set -e
...
 
 REDIS_CONTAINER_ID=$(docker run -tid -P redis:5-alpine)
 REDIS_PORT=$(docker port $REDIS_CONTAINER_ID 6379 | cut -d ':' -f 2)
 
-echo "dynamo started at: $DYNAMO_HOST"
+#echo "dynamo started at: $DYNAMO_HOST"
```
Comment (MattElcock) [inline eq-author-api/scripts/test.sh:17]:
```suggestion
```

File: eq-author-api/scripts/test.sh [hunk 5]
```diff
@@ -4,50 +4,41 @@ set -e
...
 function finish {
   echo "killing docker..."
-  docker rm -vf $DYNAMO_CONTAINER_ID
+#  docker rm -vf $DYNAMO_CONTAINER_ID
   docker rm -vf $REDIS_CONTAINER_ID
 }
 trap finish EXIT
 
-echo "waiting on Dynamo to start..."
+# echo "waiting on Dynamo to start..."
```
Comment (MattElcock) [inline eq-author-api/scripts/test.sh:27]:
```suggestion
```

File: eq-author-api/scripts/test.sh [hunk 6]
```diff
@@ -4,50 +4,41 @@ set -e
...
 
 # Poll the endpoint once a second, for 20 seconds, then fail if a successful response has not been heard.
```
Comment (MattElcock) [inline eq-author-api/scripts/test.sh:29]:
```suggestion
```

File: eq-author-api/scripts/test.sh [hunk 7]
```diff
@@ -4,50 +4,41 @@ set -e
...
-./node_modules/.bin/wait-on -d 1000 -t 20000 http://$DYNAMO_HOST/shell
+# ./node_modules/.bin/wait-on -d 1000 -t 20000 http://$DYNAMO_HOST/shell
```
Comment (MattElcock) [inline eq-author-api/scripts/test.sh:30]:
```suggestion
```

**Review by MattElcock — CHANGES_REQUESTED** [review]

**Review by MattElcock — APPROVED** [review]

**Comment by MattElcock** [comment]
This is so much better than our previous impelmentation. Not only because it's cut down the code by tonnes, it's also a lot nicer to understand. I like the tidying you've done, too, in the tests; swapping `.info` to `.debug` makes a lot of sense as it'll clear up out production output.

**Review by MattElcock — APPROVED** [review]

**Review by jasehumphr — APPROVED** [review]

**Pull request merged**